app/controllers/picks_controller.rb

```diff
@@ -19,7 +19,15 @@ def show
   end
 
   def create
-    @pick = Pick.new(pick_params)
+    @pick = Pick.create(pick_params)
```

Comment:
What about something like the following pattern:

```
    @offer = Offer.new(offer_params)

    respond_to do |format|
      if @offer.save
        format.jsonapi { render :show, status: :created, location: @offer }
      else
        format.jsonapi { render jsonapi: @offer.errors, status: :unprocessable_entity }
      end
```

But just replace Offer with pick?